log/log_test.go

```diff
@@ -0,0 +1,84 @@
+package log
```

Comment:
Why not make this `package_test` in order to have a test that cannot (even by accident) test anything else than the exported symbols?